packages/table/src/css/index.css

```diff
@@ -17,168 +19,212 @@
 }
+
 .psds-table__head.psds-theme--light {
   color: var(--ps-colors-text-icon-low-on-light);
 }
+
 .psds-table__header {
   padding: 0;
   text-align: left;
 }
-.psds-table__header > div {
+
+.psds-table__header>div {
   font-weight: var(--ps-type-font-weight-strong);
   padding: var(--ps-layout-spacing-xsmall) var(--ps-layout-spacing-small);
   vertical-align: bottom;
   white-space: nowrap;
 }
+
 .psds-table__header--align-center {
   text-align: center;
 }
+
 .psds-table__header--align-left {
   text-align: left;
 }
+
 .psds-table__header--align-right {
   text-align: right;
 }
+
 .psds-table__header--sortable {
   cursor: pointer;
 }
+
 .psds-table__header--sortable.psds-theme--dark:hover {
   color: var(--ps-colors-text-icon-high-on-dark);
 }
+
 .psds-table__header--sortable.psds-theme--light:hover {
   color: var(--ps-colors-text-icon-high-on-dark);
 }
+
 .psds-table__header__sort-icon.psds-table__header__sort-icon {
   display: inline-block;
   vertical-align: inherit;
 }
+
 .psds-table__header--sticky[scope="col"] {
   border: none;
   top: 0;
   position: -webkit-sticky;
   position: sticky;
   z-index: 2;
 }
-.psds-table__header--sticky[scope="col"] > div {
+
+.psds-table__header--sticky[scope="col"]>div {
   padding: var(--ps-layout-spacing-xsmall) var(--ps-layout-spacing-small);
 }
+
 .psds-table__header--sticky[scope="col"][data-stuck] {
   position: initial;
 }
-.psds-table__header--sticky[scope="col"][data-stuck] > div {
+
+.psds-table__header--sticky[scope="col"][data-stuck]>div {
   position: fixed;
   top: 0;
   z-index: 2;
 }
+
 .psds-table__header--sticky[scope="row"] {
   left: 0;
   position: sticky;
   z-index: 1;
 }
-.psds-table__header--sticky.psds-theme--dark[scope="col"] > div {
+
+.psds-table__header--sticky.psds-theme--dark[scope="col"]>div {
   background-color: var(--ps-colors-background-dark-1);
   box-shadow: 0 1px 0 0 #343434;
 }
+
 .psds-table__header--sticky.psds-theme--dark[scope="row"] {
   background-color: var(--ps-colors-background-dark-1);
   box-shadow: 1px 0 0 0 #343434;
 }
-.psds-table__header--sticky.psds-theme--light[scope="col"] > div {
+
+.psds-table__header--sticky.psds-theme--light[scope="col"]>div {
   background-color: var(--ps-colors-background-light-3);
   box-shadow: 0 1px 0 0 #d9d9d9;
 }
+
 .psds-table__header--sticky.psds-theme--light[scope="row"] {
   background-color: var(--ps-colors-background-light-3);
   box-shadow: 1px 0 0 0 #d9d9d9;
 }
+
 .psds-table__row.psds-theme--dark {
   border-top: 1px solid #343434;
 }
-tbody .psds-table__row.psds-theme--dark:hover > td,
-tbody .psds-table__row.psds-theme--dark:hover > th {
+
+.psds-table__row.psds-theme--dark:hover>td,
+.psds-table__row.psds-theme--dark:hover>th {
   background-color: var(--ps-colors-background-dark-2);
 }
+
 .psds-table__row.psds-theme--light {
   border-top: 1px solid #d9d9d9;
 }
-tbody .psds-table__row.psds-theme--light:hover > td,
-tbody .psds-table__row.psds-theme--light:hover > th {
+
```

Comment:
@sweeting  Relevant lines of code for the PR everything else was just css prettier